<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QMainWindow" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="titlebar_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item row="1" column="2" rowspan="7">
     <layout class="QHBoxLayout" name="cam_layout"/>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="resultLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="1" column="3" rowspan="3">
     <layout class="QVBoxLayout" name="dataLayout"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="buttonLayout"/>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="numLayout"/>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QHBoxLayout" name="infoLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="OCRLayout"/>
    </item>
    <item row="4" column="0" rowspan="4" colspan="2">
     <layout class="QHBoxLayout" name="stereoscopiccamLayout"/>
    </item>
    <item row="4" column="3" rowspan="4">
     <layout class="QVBoxLayout" name="log_layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
